### PR TITLE
[FEAT] - Tutorials update (Marco)

### PR DESCRIPTION
**Pull request by marcopeix**
Better structure and overall improvements to explanations in the following tutorials:

- long-horizon forecasting
- multiple series
- cross-validation
- anomaly detection

### CODE REVIEW

File: nbs/docs/tutorials/6_multiple_series.ipynb [hunk 1]
```diff
@@ -13,7 +13,11 @@
    "id": "752a293c-d477-45e7-93d9-23fc15a23c8f",
```
Comment (mergenthaler) [inline nbs/docs/tutorials/6_multiple_series.ipynb:13]: <p>For some reason that we still don't understand, many users think of <code>multivariate</code>  as forecast using exogenous features. I think it has to do with the intuition that the forecast procedure is not only looking at past values but other variables. This has been an ongoing source of confusion. </p><p> </p><p>Maybe I would rewrite it like this:</p><p> </p><p>Note that the forecasts are still univariate. This means that although TimeGPT is a global model, it won't consider the inter-feature relationships within the target series. However, TimeGPT does support the use of exogenous variables such as categorical variables (e.g., category, brand), numerical variables (e.g., temperature, prices), or even special holidays.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/nixtla/pull/311/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

File: nbs/docs/tutorials/9_cross_validation.ipynb [hunk 2]
```diff
@@ -5,7 +5,7 @@
    "id": "6de758ee-a0d2-4b3f-acff-eed419dd17c5",
```
Comment (mergenthaler) [inline nbs/docs/tutorials/9_cross_validation.ipynb:5]: <p>This looks good. I think a diagram would make thinks a lot easier: <a href="https://nixtlaverse.nixtla.io/statsforecast/docs/tutorials/crossvalidation.html" target="_blank">https://nixtlaverse.nixtla.io/statsforecast/docs/tutorials/crossvalidation.html</a></p><p> </p><p>Also maybe we can use some extra ideas from here: <a href="https://otexts.com/fpp3/tscv.html" target="_blank">https://otexts.com/fpp3/tscv.html</a> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/nixtla/pull/311/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

**Review by mergenthaler — APPROVED** [review]
Thanks!

**Comment by mergenthaler** [comment]
@AzulGarza: I think this is ready for your review.

**Review by AzulGarza — APPROVED** [review]
awesome! lgtm🥰

**Pull request merged**